client/lib/plans/constants.js

```diff
@@ -57,10 +63,19 @@ export const plansList = {
 		]
 	},
 
+	[ PLAN_PERSONAL ]: {
+		getStoreSlug: () => PLAN_PERSONAL,
+		availableIn: ( offer ) => offer === WPCOM,
```

Comment:
Alternatively we could have two list constants that we export:
JETPACK_ACTIVE_PLANS and WPCOM_ACTIVE_PLANS that point to plan references.